tox.ini

```diff
@@ -62,7 +62,7 @@ envlist =
     grpc_contrib-{py27,py34,py35,py36}-grpc
     httplib_contrib-{py27,py34,py35,py36}
     jinja2_contrib-{py27,py34,py35,py36}-jinja{27,28,29,210}
-    molten_contrib-{py36}-molten{07}
+    molten_contrib-{py36}-molten{070,071,072}
```

Comment:
`0.7.3` was released 3 days ago.

We might just want `molten>=0.7.0,<0.7.2` and `molten>=0.7.2,<0.8.0`

Reply:
*4 days ago